commit-graph.c

```diff
@@ -1707,7 +1707,7 @@ static void expire_commit_graphs(struct write_commit_graph_context *ctx)
 	timestamp_t expire_time = time(NULL);
 
 	if (ctx->split_opts && ctx->split_opts->expire_time)
-		expire_time -= ctx->split_opts->expire_time;
```

Comment:
One other question here because it seems this should be working since it was taking the current time in seconds and subtracting some number of seconds to create the expire time.  Was the issue just in the test and the fact that it was passing in a date instead of number of seconds?

Reply:
The `OPT_EXPIRY_DATE()` parses our seconds as "something" but it wasn't a literal integer value. It likely read our `3600` and thought of it as `3600-01-01 00:00.00`.